src/main/java/org/mozilla/jss/nss/Buffer.c

```diff
@@ -10,175 +10,168 @@
 #include "_jni/org_mozilla_jss_nss_Buffer.h"
 
 JNIEXPORT jobject JNICALL
-Java_org_mozilla_jss_nss_Buffer_Create(JNIEnv *env, jclass clazz, jlong length)
-{
-    j_buffer *buf = NULL;
+Java_org_mozilla_jss_nss_Buffer_Create(JNIEnv *env, jclass clazz, jlong length) {
+	j_buffer *buf = NULL;
```

Comment:
Same issue as in other PRs, the C code are being indented with tabs instead of 4 spaces.